docs/TOOLS.md

```diff
@@ -1,8 +1,15 @@
 # Tools
 
-A list of tools, IDEs, and editors that can help with writing and debugging Python code.
 
-<br>
+~~~~exercism/noe
```

Comment:
@BethanyG , typo? `exercism/note`

(cannot resist, sorry) I have admonished your admonition

Reply:
I have duly accepted you admonishment for the admonition accident.